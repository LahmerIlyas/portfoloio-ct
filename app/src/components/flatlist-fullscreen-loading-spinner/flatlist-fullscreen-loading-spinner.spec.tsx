import React from 'react';
import renderer from 'react-test-renderer';
import { FlatListFullscreenLoadingSpinner } from '.';

describe('<FlatListFullscreenLoadingSpinner />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<FlatListFullscreenLoadingSpinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
