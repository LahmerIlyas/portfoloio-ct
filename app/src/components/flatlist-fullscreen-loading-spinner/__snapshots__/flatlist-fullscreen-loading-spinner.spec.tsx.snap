// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlatListFullscreenLoadingSpinner /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#91C788"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
